etc/keystrokes.cfg

```diff
@@ -8,10 +8,6 @@
 <keystroke action="defer" Control="1" AltMeta="1" key="e" />
 <keystroke action="ignore" Control="1" AltMeta="1" key="BackSpace" />
 <keystroke action="force_synchronization" Control="1" AltMeta="1" key="j" />
-<keystroke action="debug_tree" Control="1" AltMeta="1" key="q" />
-<keystroke action="regions_on_screen" Control="1" AltMeta="1" key="a" />
-<keystroke action="test_input" Control="1" AltMeta="1" key="x" />
-<keystroke action="deactivate_input_devices_grab" Control="1" AltMeta="1" key="y" />
```

Comment:
They are docuemented. I doubt that having them commented in the keystrokes.cfg (which really should come as an example only) will be beneficial for anyone installing the package because the binary will still not support those keys. So the user must compile himself. For that case we could add an keystrokes.cfg.debug which does not get packaged.